utilities/getArticleByTitle/index.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { ArticleProps } from '../../components/Article'
+import { CONTENTFUL_API_URL } from '../constants'
+import getArticleByTitle from './index'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import 'whatwg-fetch'
+
+const mockTitle = 'mock title'
+const article: ArticleProps = {
+  title: mockTitle,
+  content: 'mock content',
+}
+const server = setupServer(
+  rest.post(CONTENTFUL_API_URL, (req, res, ctx) => {
+    return res(
+      ctx.json({
+        data: {
+          articleCollection: {
+            items: [article],
+          },
+        },
+      }),
+    )
+  }),
+)
+
+beforeAll(() => server.listen({ onUnhandledRequest: 'error' }))
+afterAll(() => server.close())
+afterEach(() => server.resetHandlers())
+
+describe('a get article test', () => {
```

Comment:
I'm pretty sure that Dodds has given up on the `describe` blocks and I'm happy to just go with `test`. It used to be that the `describe` blocks printed out nicely (maybe that was mocha?), but for years I haven't seen any benefit to them and have just done them by habit. Maybe it's time to break the habit. So this might be:

```typescript
test('get page content by slug', () => { ... }
```

Reply:
OK